static/install.sh

```diff
@@ -42,6 +44,42 @@ errorevent() {
 
 trap "errorevent" ERR
 
+# Compares two version numbers.
+# Returns 0 if the versions are equal, 1 if the first version is higher, and 2 if the second version is higher.
+compare_version () {
```

Comment:
let's add the original link for the code we copied over, for whoever would change this in the future.
https://stackoverflow.com/questions/4023830/how-to-compare-two-strings-in-dot-separated-version-format-in-bash

also that comment has some tests, would be neat to run them as well, in case we change this function in the future.

Reply:
good point